core/templates/dev/head/galleries/Gallery.js

```diff
@@ -203,6 +203,125 @@ oppia.controller('Gallery', [
       $rootScope.loadingMessage = '';
     });
 
+    // Below is the width of each tile
+    var TILEDISPLAYWIDTH = 211.953;
+    // If the value below is changed, the corresponding CSS values must be
+    // changed in oppia.css.
+    var MAXTILEDISPLAY = 4;
+    var maxGalleryTileWidth = TILEDISPLAYWIDTH * MAXTILEDISPLAY;
```

Comment:
I don't think you need this variable at all; you can calculate it on the fly based on the number of displayed tiles.